### PR TITLE
Handle the case when issue labels already exist

### PR DESCRIPTION
**Pull request by sethvargo**
This fixes GH-13163

I'm not sure how to write an acceptance test for this, since I'd have to create the issue outside of the acceptance test first. Any suggestions or examples of where we do that already?

### CODE REVIEW

**Comment by radeksimko** [comment]
Existing acceptance tests are passing, it would be nice to add another one covering this use case.
The code otherwise looks good, assuming conflicts are resolved.

> I'd have to create the issue outside of the acceptance test first

How about we just create `github_repository` as part of the acceptance test which will have the mentioned default labels and then try modified one of the default labels we expect to exist there?

File: builtin/providers/github/resource_github_issue_label_test.go [hunk 1]
```diff
@@ -134,3 +141,16 @@ resource "github_issue_label" "test" {
   color      = "FFFFFF"
 }
 `, testRepo)
+
+var testAccGitHubIssueLabelExistsConfig string = fmt.Sprintf(`
+// Create a repository which has the default labels
+resource "github_repository" "test" {
+  name = "tf-acc-repo-label-abc1234"
```
Comment (radeksimko) [inline builtin/providers/github/resource_github_issue_label_test.go]: I think we'll need to randomize this, but apparently the other tests here aren't randomized yet either, so I'll do it in a separate PR.

File: website/source/docs/providers/github/r/issue_label.html.markdown [hunk 2]
```diff
@@ -6,16 +6,24 @@ description: |-
   Provides a GitHub issue label resource.
 ---
 
-# github\_issue_label
+# github_issue_label
```
Comment (radeksimko) [inline website/source/docs/providers/github/r/issue_label.html.markdown]: I never really understood the point of the `\` escaping here in this context 🤔

File: builtin/providers/github/resource_github_issue_label_test.go (hunk 1 above)
Comment (sethvargo) [inline builtin/providers/github/resource_github_issue_label_test.go]: Yea, I tried that. I used random_id, but random_id doesn't get compiled 😦

File: website/source/docs/providers/github/r/issue_label.html.markdown (hunk 2 above)
Comment (sethvargo) [inline website/source/docs/providers/github/r/issue_label.html.markdown]: It does nothing... Trust me lol. This is identical.

File: builtin/providers/github/resource_github_issue_label_test.go (hunk 1 above)
Comment (radeksimko) [inline builtin/providers/github/resource_github_issue_label_test.go]: So we usually generate the randomness within the test via helpers (https://github.com/hashicorp/terraform/tree/master/helper/acctest), intentionally to not couple providers like `random` with other ones.

**Pull request merged**